iampolicymanagementv1/iam_policy_management_v1_integration_test.go

```diff
@@ -1500,6 +1512,412 @@ var _ = Describe("IAM Policy Management - Integration Tests", func() {
 		})
 	})
 
+	Describe(`CreateBasicActionControlTemplate - Create a basic action template version without action_control`, func() {
+		BeforeEach(func() {
+			shouldSkipTest()
+		})
+		It(`CreateBasicActionControlTemplate(createActionControlTemplateOptions *CreateActionControlTemplateOptions) without action_control`, func() {
+			createActionControlTemplateOptions := &iampolicymanagementv1.CreateActionControlTemplateOptions{
+				Name:           core.StringPtr(exampleBasicActionControlTemplateName),
+				AccountID:      &testTargetEnterpriseID,
+				Description:    core.StringPtr("Test Basic ActionControl Template from GO SDK"),
+				AcceptLanguage: core.StringPtr("default"),
+			}
+
+			actionControlTemplate, response, err := service.CreateActionControlTemplate(createActionControlTemplateOptions)
+			Expect(err).To(BeNil())
+			Expect(response.StatusCode).To(Equal(201))
+			Expect(actionControlTemplate).ToNot(BeNil())
+			Expect(actionControlTemplate.Name).To(Equal(core.StringPtr(exampleBasicActionControlTemplateName)))
+			Expect(actionControlTemplate.AccountID).To(Equal(core.StringPtr(testTargetEnterpriseID)))
+			Expect(actionControlTemplate.State).To(Equal(core.StringPtr("active")))
+
+			testBasicActionControlTemplateID = *actionControlTemplate.ID
+			testBasicActionControlTemplateETag = response.GetHeaders().Get(etagHeader)
+			testBasicActionControlTemplateVersions = *actionControlTemplate.Version
+		})
+	})
+
+	Describe(`ReplaceBasicActionControlTemplate - Update a basic action control template version without action_control`, func() {
+		BeforeEach(func() {
+			shouldSkipTest()
+		})
+		It(`ReplaceActionControlTemplate(replaceActionControlTemplateOptions *ReplaceActionControlTemplateOptions)`, func() {
+			replaceActionControlTemplateOptions := &iampolicymanagementv1.ReplaceActionControlTemplateOptions{
+				Description:             core.StringPtr("Test ActionControl Template from GO SDK"),
+				Name:                    core.StringPtr(exampleBasicActionControlUpdateTemplateName),
+				ActionControlTemplateID: core.StringPtr(testBasicActionControlTemplateID),
+				IfMatch:                 core.StringPtr(testBasicActionControlTemplateETag),
+				Version:                 core.StringPtr(testBasicActionControlTemplateVersions),
+			}
+
+			actionControlTemplate, response, err := service.ReplaceActionControlTemplate(replaceActionControlTemplateOptions)
```

Comment:
Do we need two integration tests for Replace method?

Reply:
Added an additional check to detect changes in the basic template (without action_control).